app/api/routes.py

```diff
@@ -25,6 +28,27 @@ def get_plant(id):
     response.status_code = 200
     return response
 
+@api.route('/garden', methods=['POST'])
```

Comment:
Our first Post route!  I'm impressed how smoothly it went.  I also like that we're moving the plant names from the plant table to this table, makes a lot more sense.